src/rustllvm/PassWrapper.cpp

```diff
@@ -188,7 +188,7 @@ LLVMRustCreateTargetMachine(const char *triple,
     }
 
     TargetOptions Options;
-    Options.PositionIndependentExecutable = PositionIndependentExecutable;
+    //Options.PositionIndependentExecutable = PositionIndependentExecutable;
```

Comment:
I'm pretty sure this is a hack that only makes sense in my fork (there's no PIE for microcontrollers :-)).

Even if we wanted to remove this (doubtful), it should be removed and not commented-out. I believe that LLVM applies this flag at a more granular level now, so looking downward for another flag to apply it to seems reasonable.